### ISSUE TITLE
MB: QoL: Seed the batch recording comments script v2021.6.5

### ISSUE THREAD

**Issue by tigerman325** [ROpdebee/mb-userscripts#24]
One little small error. Guidelines state to use USA for United States, not just US.
https://musicbrainz.org/doc/Style/Specific_types_of_releases/Live_bootlegs.

Also, you can't add live disambiguation without going through the release and attaching a work to every recording set to live? Shouldn't the live release group type be enough? I know of no other way to mark a relationship as live except for the work - recording relationship and I get errors saying that it skips all recordings not set to live.

**Comment by ROpdebee** [comment]
Both issues should be fixed.
The USA part was an oversight.

About the live recordings: Originally it didn't perform any checks to make sure the recordings were actually live. Then while testing for #21 I noticed it was setting comments on recordings that weren't live, so I implemented those work AR attribute checks. They are indeed a bit conservative. The new checks will consider a recording to be live if either
1) There is >= 1 work AR for the recording with the live attribute set
2) There are no work ARs for the recording and the RG is live
There are certainly still cases where that check will pass where it shouldn't, and vice versa, but they should be relatively easy to manually fix and hopefully quite rare.
